components/automate-ui/src/styles/_colors.scss

```diff
@@ -51,16 +51,17 @@ $blue: #174AF0;
 $malibu: #648FFF;
 // secondary
 $pink: #DC267F;
-$violet: #C22DD5;
+$violet: #A424B4;
 $indigo: #785EF0;
 $science-blue: #0075DB;
+$medium-teal-blue: #005BAB;
```

Comment:
this was in the ui library but not here, so i added it.

Reply:
Not sure this is appropriate and/or it is a bigger bucket of fish to fry. We are importing the styles from the chef-ui-library here:
https://github.com/chef/automate/blob/4ed76448b0d9e9298b2c0b6690eded29f9830341/components/automate-ui/src/styles.scss#L5
So what you have done is technically redundant. So maybe we do not want/need to import styles from chef-ui-library. But then there are probably other colors that have been duplicated this way, too. 🤷 No idea what the right answer is here.